### PR TITLE
[PL-45695] SMP Install Istio

### PR DESCRIPTION
**Pull request by brian-r-fisher**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: SMP Install Istio
* Jira/GitHub Issue numbers (if any): PL-45695
* Preview links/images (Internal contributors only): [Configure and deploy Istio](https://66465e7468f55205a770c9f6--harness-developer.netlify.app/docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio)

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+type: Opaque
+```
+</details>
+
+### Install Istio
+
+To install Istio, do the following:
+
+1. Run the following command to download the latest version of Istio.
+
```
Comment (SmritiSatya) [inline docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md:196]: Download the latest version of Istio.

File: docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+metadata:
+  name: istio-gw-tls
+  namespace: istio-system
+type: Opaque
+```
+</details>
+
+### Install Istio
+
+To install Istio, do the following:
```
Comment (SmritiSatya) [inline docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md:193]: To install Istio, perform the following steps:

File: docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+   ```
+   cd istio-1.xx.x
+   ```
+
+   ```
+   export PATH=$PWD/bin:$PATH
+   ```
+
+3. Run the following to validate installation.
```
Comment (SmritiSatya) [inline docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md:211]: Validate the installation.

File: docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+   istioctl version
+   ```
+
+4. Open the `istio-operator.yaml` file, and go to the service annotations on line 44.
+
+5. On line 46, copy and paste the Elastic IPs created for Istio. The number of Elastic IPs depends on the number of private subnets in your cluster.
+
+6. Import SSL cert, key and cacert in AWS certificate manager and crate new certificate.
+
+7. Copy the arn of new cert to line 48
```
Comment (SmritiSatya) [inline docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md:223]: 7. Copy the ARN of new cert to line 48.

File: docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+   ```
+   istioctl version
+   ```
+
+4. Open the `istio-operator.yaml` file, and go to the service annotations on line 44.
+
+5. On line 46, copy and paste the Elastic IPs created for Istio. The number of Elastic IPs depends on the number of private subnets in your cluster.
+
+6. Import SSL cert, key and cacert in AWS certificate manager and crate new certificate.
```
Comment (SmritiSatya) [inline docs/self-managed-enterprise-edition/advanced-configurations/configure-and-deploy-istio.md:221]: Import SSL cert, key and cacert in AWS certificate manager and create a new certificate.

**Review by SmritiSatya — APPROVED** [review]

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**